common/core/desktop/src/kmx/kmx_file.cpp

```diff
@@ -238,8 +238,8 @@ LPKEYBOARD KMX_Processor::CopyKeyboard(PKMX_BYTE bufp, PKMX_BYTE base)
     i < kbp->cxGroupArray;
     i++, gp++, cgp++)
   {
-    gp->dpName = (PKMX_WCHAR)(base + cgp->dpName);
-    gp->dpKeyArray = (LPKEY) bufp;
+    gp->dpName = StringOffset(base, cgp->dpName);
```

Comment:
Is part of this cleanup because `StringOffset` handles all the casting?
`PKMX_WCHAR`, 
`PWCHAR` (K32_load l.732)
`PWSTR` (K32_load l.800)

Reply:
`StringOffset` handles `NULL` properly :)